src/tools/compiletest/src/common.rs

```diff
@@ -245,6 +245,11 @@ pub struct Config {
     /// mode describing what file the actual ui output will be compared to
     pub compare_mode: Option<CompareMode>,
 
+    /// If true, this will generate a coverage file with UI test files that run `MachineApplicable`
+    /// lints but are missing `run-rustfix` annotations. The generated coverage file is created in
```

Comment:
```suggestion
    /// diagnostics but are missing `run-rustfix` annotations. The generated coverage file is created in
```
Not all suggestions come from lints.